src/Nancy.Testing.Tests/BrowserFixture.cs

```diff
@@ -318,15 +322,13 @@ public void Should_be_able_to_continue_with_another_request()
         }
 
         [Fact]
-        public void Should_maintain_cookies_when_chaining_requests()
+        public async Task Should_maintain_cookies_when_chaining_requests()
         {
             // Given
             // When
-            var result = browser.Get(
-                             "/session",
-                             with => with.HttpRequest())
-                .Then

```

Comment:
extensions methods around awaited methods = fugly. Have to surround the method in more brackets first:

```
(await browser.Get("/session", with => with.HttpRequest())).Then()
```